packages/daemon/src/daemon.js

```diff
@@ -227,6 +227,26 @@ const makeEndoBootstrap = (
     return { external, internal: undefined };
   };
 
+  /**
+   * @param {string} agentFormulaIdentifier
+   * @param {string[]} path
+   * @param {import('./types.js').Terminator} terminator
+   */
+  const makeControllerForLookup = async (
+    agentFormulaIdentifier,
+    path,
+    terminator,
+  ) => {
+    terminator.thisDiesIfThatDies(agentFormulaIdentifier);
+
+    // Behold, recursion:
+    // eslint-disable-next-line no-use-before-define
+    const agent = provideValueForFormulaIdentifier(agentFormulaIdentifier);
```

Comment:
Do we want `lookup` to work for all the formula types that contain a pet store? That’d include `host`, `guest`, and eventually `directory` if not others. We could insist that the formula identifier corresponds to the underlying `pet-store` (which would simplify the controller maker for watching the pet store for changes and propagating cancellation. That would put the responsibility for looking up the pet store higher up.

Reply:
Use either `hub` or `store`, not `agent`. Revert use of `pet-store` formula identifier to create `lookup` formula identifier.